src/translations/en.json

```diff
@@ -7161,6 +7161,29 @@
         "next": "Next",
         "finish": "Finish",
         "help": "Help",
+        "prepare": {
```

Comment:
I'm surprised this is under `page-onboarding`. Shouldn't it be under a `page-preparing` ? It's a standalone app from the onboarding app.